developer/src/tike/main/Keyman.System.KeymanCore.pas

```diff
@@ -201,6 +205,35 @@   km_core_action_item = record
 
   pkm_core_action_item = ^km_core_action_item;
 
+  km_core_bool = uint32;
```

Comment:
Could this be an enum too?
https://github.com/keymanapp/keyman/blob/c494581117b76ac1453aa375faa9bbb99330cf86/core/include/keyman/keyman_core_api.h#L574

```suggestion
  km_core_bool = (
    KM_CORE_FALSE = 0,
    KM_CORE_TRUE = 1
  );
```

Reply:
Unfortunately we can't really do that because then it becomes a uint8 internally.